src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -76,6 +77,40 @@ def get_shapefile_crs(data_source):
     return epsg_code
 
 
+def crop_raster(input_raster, output_raster_file, polygon_extent, dest_crs):
+    '''
+    Crops the image specified by input_raster using the limits defined in
+    polygon_extent (buffer already applied).
+
+    input_raster: path to the image to be cropped.
+    output_raster_file: the output file path.
+    polygon_extent: POLYGON object to be used for the crop limits.
+    dest_crs: the CRS of the output raster.
+    '''
+    bounds = polygon_extent.GetEnvelope()
+    command_opts = str(bounds[0]) + ' ' + str(bounds[2]) + ' ' \
+        + str(bounds[1]) + ' ' + str(bounds[3])
+    gdal_command = 'gdalwarp' + ' -t_srs EPSG:' + str(dest_crs) + ' -te ' \
+                   + str(command_opts) + ' ' + input_raster \
+                   + ' ' + output_raster_file
+    try:
+        process = subprocess.Popen(gdal_command,
```

Comment:
Perhaps we should see this with @91nunocosta. We are repeating code here to execute the command line.